app/views/friends/index.html.erb

```diff
@@ -3,9 +3,12 @@
   <ul>
     <% @friends.each do |friend| %>
       <li>
+        debugger
         <%= friend.name %> (email: <%= friend.email %>)
-        <%= form_for friend, url: friendship_path(friend), method: :delete, html: { class: "friendship-form" } do |f| %>
-          <%= f.submit "Remove Friendship", class: "btn btn-danger", data: { confirm: "Are you sure you want to remove this friendship?" } %>
+        <%= form_with model: Friendship.find_by(user_id: current_user.id, friend_id: friend.id),
```

Comment:
You may not need to hit the database to define `model` here, because you define the `url:` below:
```suggestion
        <%= form_with
```